src/main/java/ch/njol/skript/util/SkriptColor.java

```diff
@@ -94,50 +125,42 @@ public ChatColor asChatColor() {
 	public DyeColor asDyeColor() {
 		return dye;
 	}
-	
+
 	@Override
 	public String getName() {
-		return name();
+		return Streams.findLast(names.entrySet().stream()
```

Comment:
I'd rather make the Map a BiMap and then use `names.inverse().get(this)`, the current method seems overly complex for what it does.

Reply:
It's more efficient using Streams FindTasks than inverting the whole list.

Reply:
I completely doubt that after looking at findLast's implementation, and BiMap doesn't exactly reverse the whole list, it's a pretty efficient collection

Reply:
http://hg.openjdk.java.net/jdk8/jdk8/jdk/file/687fd7c7986d/src/share/classes/java/util/stream

Reply:
Not sure what does the jdk repo has to do with anything? You're using Guava's Streams util there.

Reply:
It's the source code for Java 8's jdk

Reply:
It's just not going to work out in my opinion.